packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -300,6 +301,9 @@ class TextStyle extends Diagnosticable {
   /// The locale used to select region-specific glyphs.
   final ui.Locale locale;
```

Comment:
while you're at it, let's export Locale from basic_types too so we don't need to use `ui.` here either